model/textparse/interface_test.go

```diff
@@ -104,3 +111,90 @@ func TestNewParser(t *testing.T) {
 		})
 	}
 }
+
+// parsedEntry represents data that is parsed for each entry.
+type parsedEntry struct {
+	// In all but EntryComment, EntryInvalid.
+	m string
+
+	// In EntryHistogram.
+	shs *histogram.Histogram
+	fhs *histogram.FloatHistogram
+
+	// In EntrySeries.
+	v float64
+
+	// In EntrySeries and EntryHistogram.
+	lset labels.Labels
+	t    *int64
+	es   []exemplar.Exemplar
+	ct   *int64
```

Comment:
```suggestion
	metric string

	// In EntryHistogram.
	shs *histogram.Histogram
	fhs *histogram.FloatHistogram

	// In EntrySeries.
	value float64

	// In EntrySeries and EntryHistogram.
	labelSet labels.Labels
	timestamp    *int64
	exemplarsSet   []exemplar.Exemplar
	createdtimestamp   *int64
```

I am not sure how everybody else feels about it, but I think having more descriptive variable names helps folks who aren't used to the codebase understand it faster.

No strong opinions, feel free to ignore if you prefer :)